src/view/com/composer/Composer.tsx

```diff
@@ -129,6 +130,16 @@ export const ComposePost = observer(function ComposePost({
   const {closeAllDialogs} = useDialogStateControlContext()
   const t = useTheme()
 
+  // Disable this in the composer to prevent any extra keyboard height being applied.
+  // See https://github.com/bluesky-social/social-app/pull/4399
+  const context = useKeyboardContext()
+  React.useEffect(() => {
+    context.setEnabled(false)
+    return () => {
+      context.setEnabled(true)
+    }
+  }, [context])
```

Comment:
maybe destructure setEnabled outside and depend just on that? if that works. just in case.